### PR TITLE
Fixed some class tests

### PR DESCRIPTION
**Pull request by tanay-man**

### CODE REVIEW

**Comment by certik** [comment]
The change seems fine. The CI currently fails.

**Comment by tanay-man** [comment]
> The change seems fine. The CI currently fails.

Yeah I am not sure why this is the case. @Thirumalai-Shaktivel can you review this too?

**Comment by certik** [comment]
You have to update reference tests.

**Comment by tanay-man** [comment]
This PR is breaking the following integration tests using llvm11 :
```
632 - modules_24 (Not Run)
635 - modules_38 (Not Run)
639 - modules_47 (Not Run)
690 - interface_12a (Not Run)
696 - generic_name_01 (Not Run)
702 - operator_overloading_06 (Not Run)
747 - derived_types_08 (Not Run)
750 - derived_types_12 (Not Run)
752 - derived_types_15 (Not Run)
756 - derived_types_24 (Not Run)
783 - allocate_13 (Not Run)
801 - associate_08 (Not Run)
909 - class_01 (Not Run)
910 - class_02 (Not Run)
913 - class_05 (Not Run)
916 - class_08 (Not Run)
917 - class_09 (Not Run)
918 - class_10 (Not Run)
919 - class_11 (Not Run)
920 - class_12 (Not Run)
981 - template_vector (Not Run)
```
and the following tests :
```
llvm-class_01
llvm-class_02
llvm-generic_name_01
llvm-modules_36
llvm-modules_38
```

**Comment by certik** [comment]
Just take one failing test, and run it directly and investigate why it fails.

File: integration_tests/run_tests.py
Comment (advikkabra) [inline integration_tests/run_tests.py:1]: You'll have to unstage this, I think you've staged it by mistake

**Review by Thirumalai-Shaktivel — APPROVED** [review]

**Pull request merged**